lib/elixlsx/sheet.ex

```diff
@@ -205,4 +214,76 @@ defmodule Elixlsx.Sheet do
   def remove_pane_freeze(sheet) do
     %{sheet | pane_freeze: nil}
   end
+
+
+  @spec set_autofilter(Sheet.t(), String.t(), String.t()) :: Sheet.t()
+  @doc ~S"""
+  Set the range for autofiltering
+  """
+  def set_autofilter(sheet, start_cell, end_cell) do
+    {row1, col1} = Util.from_excel_coords(start_cell)
+    {row2, col2} = Util.from_excel_coords(end_cell)
+    set_autofilter(sheet, row1, col1, row2, col2)
+  end
+
+  @spec set_autofilter(Sheet.t(), number, number, number, number) :: Sheet.t()
+  @doc ~S"""
+  Set the range for autofiltering
+  """
+  def set_autofilter(sheet, row1, col1, row2, col2) do
+    %{sheet | autofilter_ref: {row1, col1, row2, col2}}
+  end
+
+
+  @spec append_list_filter(Sheet.t(), String.t(), list(any())) :: Sheet.t()
+  @doc ~S"""
+  Add filter on a column as a list of inclusive elements
+  """
+  def append_list_filter(sheet, column, filters) when is_binary(column) do
+    append_list_filter(sheet, Util.decode_col(column), filters)
+  end
+
+  @spec append_list_filter(Sheet.t(), number, list(any())) :: Sheet.t()
+  @doc ~S"""
+  Add filter on a column as a list of inclusive elements
+  """
+  def append_list_filter(sheet, column, filters) do
+    update_in(sheet.autofilter_cols, &Map.put(&1, column, {:list, filters}))
+  end
+
+
+  @spec append_criteria_filter(Sheet.t(), String.t(), operator, any()) :: Sheet.t()
+  @doc ~S"""
+  Add filter on a column as an operator-based criteria
```

Comment:
Might make sense to document the supported operators somewhere

Reply:
It is documented now.